pkg/driver/node.go

```diff
@@ -66,6 +67,7 @@ type nodeService struct {
 	driverOptions *Options
 	pvmInstanceId string
 	volumeLocks   *util.VolumeLocks
+	stats         StatsUtils
```

Comment:
Don't we need to declare stats as a pointer?
it is assigned with address as below
```
stats:         &VolumeStatUtils{},
```

Reply:
Here StatsUtils is an interface and receiver methods of this interface receives a pointer so used &VolumeStatUtils{}.